extensions/vertx-web/deployment/src/main/java/io/quarkus/vertx/web/deployment/HandlerDescriptor.java

```diff
@@ -31,6 +35,34 @@ boolean isReturningMulti() {
         return method.returnType().name().equals(DotNames.MULTI);
     }
 
+    /**
+     * @return {@code true} if the method is annotated with a constraint or {@code @Valid} or any parameter has such kind of
+     *         annotation.
+     */
+    boolean requireValidation() {
+        return method.annotations().stream()
+                .anyMatch(new Predicate<AnnotationInstance>() {
+                    @Override
+                    public boolean test(AnnotationInstance ai) {
+                        return ai.name().toString().startsWith("javax.validation");
```

Comment:
Hmmm, no, that won't work :). We have `org.hibernate` specific constraints and users develop custom constraints.

Maybe I should add a build item that can send you the list of constraints?

Reply:
BTW I also like streams but a loop returning `true` once a constraint annotation is found would be more readable ;-)

Reply:
I've changed the code to use regular loops. 

About the name, I've done a temporary fix until we get the built item.